its/ruling/src/test/resources/guava/squid-S4144.json

```diff
@@ -0,0 +1,68 @@
+{
+'com.google.guava:guava:src/com/google/common/base/CharMatcher.java':[
+1094,
```

Comment:
I am aslo doubtful about those : overriding method from interface with same implementation, not sure the suggested fix will actually be nice.